### PR TITLE
Notification

### PR DESCRIPTION
**Pull request by QuentinBouygues**
It works, but not to be merged now. I prefer to wait the end of the front page to merge it all at once.

Updated routes, and automated notif creation : A notif is created when a board is updated for each user on the board

### CODE REVIEW

File: src/bl/boardFacade.ts [hunk 1]
```diff
@@ -66,12 +68,14 @@ export class BoardFacade {
         }
     }
 
-    static async update(params: {}, boardId: number): Promise<Board> {
+    static async update(params: express.Request, boardId: number): Promise<Board> {
```
Comment (Porquepix) [inline src/bl/boardFacade.ts]: Facade should not get any express stuff. You need to give the the body and the requester.
`static async update(requester: Requester, boardId: number, params: {}): Promise<Board> {`

File: src/bl/notificationFacade.ts [hunk 2]
```diff
@@ -1,23 +1,26 @@
-import { getManager } from 'typeorm'
+import { getManager, getRepository } from 'typeorm'
```
Comment (Porquepix) [inline src/bl/notificationFacade.ts]: getManager should be useless.

File: src/bl/notificationFacade.ts [hunk 3]
```diff
@@ -49,11 +44,35 @@ export class NotificationFacade {
     static async create(notification: Notification): Promise<Notification> {
         try {
             let notificationToCreate = new Notification()
-            notificationToCreate = ParamsExtractor.extract<Notification>(['about', 'from', 'type', 'user'],
+            notificationToCreate = ParamsExtractor.extract<Notification>(['about', 'from', 'type', 'user', 'date'],
```
Comment (Porquepix) [inline src/bl/notificationFacade.ts]: use paramsextractor v2 and fill

**Comment by QuentinBouygues** [comment]
@Porquepix Tu peux merge ? Ton test sur user register pete

**Pull request merged**